ui/nuxeo-document-picker/nuxeo-document-picker.js

```diff
@@ -0,0 +1,236 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '../search/nuxeo-results-view.js';
+import '../widgets/nuxeo-dialog.js';
+import '@polymer/paper-button/paper-button.js';
+
+{
+  /**
+   * `nuxeo-document-picker`
+   * @group Nuxeo UI
+   * @element nuxeo-document-picker
+   */
+  class DocumentPicker extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style include="nuxeo-styles">
+          :host {
+            --nuxeo-document-picker-dialog-max-height: 84%;
+            --nuxeo-document-picker-dialog-max-width: 768px;
+            --nuxeo-document-picker-results-view-height: calc(100vh - 520px - var(--nuxeo-app-top));
+          }
+          nuxeo-dialog {
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            height: 100%;
+            width: 100%;
+            max-height: var(--nuxeo-document-picker-dialog-max-height);
+            max-width: var(--nuxeo-document-picker-dialog-max-width);
```

Comment:
We should set the fallback value in the var instead:
```suggestion
            max-height: var(--nuxeo-document-picker-dialog-max-height, 84%);
            max-width: var(--nuxeo-document-picker-dialog-max-width, 768px);
```